### PR TITLE
Draw: set feature's color individually + delete features individually

### LINKED ISSUE #2421: draw tool - same color for all drawings
### What is the bug?

when the user changes the color of a picture, all the pictures take on that same color

### Steps to reproduce the issue

try to change color

### Lizmap version

3.4.4

### QGIS desktop version

3.16.8

### QGIS server version

3.16.8

### Operating system

Ubuntu 20.04

### Browsers

Firefox

### Browsers version

Firefox 89

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by nboisteault**
Funded by [Lons-le-Saunier](https://www.lonslesaunier.fr/)

https://user-images.githubusercontent.com/2145040/224732868-889e7e50-932b-4bce-94c0-1c1c4874b973.mp4

Fix #2421

### CODE REVIEW

**Comment by rldhont** [comment]
Tests ?

File: lizmap/modules/view/locales/en_US/dictionnary.UTF-8.properties [hunk 1]
```diff
@@ -199,7 +199,7 @@ digitizing.toolbar.polygon=Polygon
 digitizing.toolbar.freehand=Free hand
 digitizing.toolbar.import=Import in GeoJSON, GPX or KML
 digitizing.toolbar.erase=Delete all features
```
Comment (Gustry) [inline lizmap/modules/view/locales/en_US/dictionnary.UTF-8.properties:201]:
```suggestion
digitizing.toolbar.erase=Delete some features
```

The user is free to remove one or many with this label ?

**Review by Gustry — COMMENTED** [review]
I find it hard to see if the feature is selected or not. Possible to show like in QGIS some nodes from the geometry highlighted ? Open to discuss

I would add in the "Edit" tooltip saying that multiple features can be selected with "shift" ?

Is this white space needed at the bottom ?
image

image

**Comment by nboisteault** [comment]
> I find it hard to see if the feature is selected or not. Possible to show like in QGIS some nodes from the geometry highlighted ? Open to discuss
> 
> I would add in the "Edit" tooltip saying that multiple features can be selected with "shift" ?
> 
> Is this white space needed at the bottom ? image
> 
> image

What about `Edit (hold "Shift" to select multiple features)`

**Comment by Gustry** [comment]
> @Gustry What about Edit (hold "Shift" to select multiple features)?

Yes

Side note, can you add tooltip on the "Distance" label and widget (same for "angle"), explaining the concept ? I think while drawing rectangle, these settings are not needed, no ?

**Review by Gustry — APPROVED** [review]
I didn't try this new one, but from my last comments, it seems solved ?
Good

**Pull request merged**